src/Symfony/Component/Console/Input/InputInterface.php

```diff
@@ -59,6 +59,13 @@ public function getParameterOption($values, $default = false);
 
     /**
      * Binds the current Input instance with the given arguments and options.
+     *
+     * @throws RuntimeException When one of the following conditions is fulfilled:
```

Comment:
I'm not sure about giving all these internal details here ... because they may change in the future and it's easy to forget to update this comment.

Reply:
I agree, and it's implementation specific. Changed to a simple `@throws RuntimeException`